lib/vehicle.js

```diff
@@ -494,4 +484,21 @@ Vehicle.prototype.unlock = function() {
   return util.wrap(req).then(({status}) => ({status}));
 };
 
+/**
+ * POST Vehicle.batch
```

Comment:
Can we add here some examples of the return object and the accepted params?
```
@params {string[]} paths - A list of paths of endpoints to send requests to.
@example 
{
  responses: [....]
}

```

Reply:
added